src/widgets/biometrics/biometrics-chart.component.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import styles from "./biometrics-chart.component.scss";
+import { LineChart } from "@carbon/charts-react";
+import { useConfig } from "@openmrs/esm-react-utils";
+import { RadioButton, RadioButtonGroup } from "carbon-components-react";
+import { PatientBiometrics } from "./biometrics-overview.component";
+import { LineChartOptions } from "@carbon/charts/interfaces/charts";
+import { ScaleTypes } from "@carbon/charts/interfaces/enums";
+import dayjs from "dayjs";
+
+interface BiometricsChartProps {
+  patientBiometrics: Array<PatientBiometrics>;
+  conceptsUnits: Array<string>;
+}
+
+interface biometricChartData {
+  title: string;
+  value: number | string;
+  groupName: "weight" | "height" | "bmi" | string;
+}
+
+const BiometricsChart: React.FC<BiometricsChartProps> = ({
+  patientBiometrics,
+  conceptsUnits
+}) => {
+  const config = useConfig();
+  const { bmiUnit } = config.biometrics;
+  const [, , , heightUnit, weightUnit] = conceptsUnits;
+  const [selectedBiometrics, setSelectedBiometrics] = React.useState<
+    biometricChartData
+  >({
+    title: `Weight (${weightUnit})`,
+    value: "weight",
+    groupName: "weight"
+  });
+
+  const chartData = React.useMemo(
+    () =>
+      patientBiometrics.map(biometric => {
+        return {
+          group: selectedBiometrics.groupName,
+          key: dayjs(biometric.date).format("DD-MMM"),
+          value: biometric[selectedBiometrics.value]
+        };
+      }),
+    [patientBiometrics, selectedBiometrics]
+  );
+
+  const chartColors = React.useMemo(() => {
```

Comment:
`chartColors ` is constant, you don't need to use `useMemo` here, just move them outside of the render function (above line 22)